pkg/services/provisioning/alerting/testdata/mute_times/multiple-mute-times/mute_times.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: 1
+muteTimes:
+  - name: test
+    time_intervals:
+    - times:
+      - start_time: '06:00'
+        end_time: '23:59'
+      weekdays:
+      - monday
+  - orgId: 1337
+    name: test
+    time_intervals:
+    - times:
+      - start_time: '06:00'
+        end_time: '23:59'
+      weekdays:
```

Comment:
We should have at least one test for mute timings that contains all the possible ranges/fields (day of week, month, year, etc). The resulting mute timing looks really weird but I ran into **several** problems here in Terraform that this caught, and would have been un-caught otherwise.

Reply:
Added a test with all accepted inputs by upstream.